salt/utils/zeromq.py

```diff
@@ -82,6 +82,8 @@ def ip_bracket(addr):
     Convert IP address representation to ZMQ (URL) format. ZMQ expects
     brackets around IPv6 literals, since they are used in URLs.
     '''
-    if addr and ':' in addr and not addr.startswith('['):
-        return '[{0}]'.format(addr)
+    _ip_addr = ipaddress.ip_address(addr)
+    if _ip_addr.version == 6:
+        return '[{0}]'.format(str(_ip_addr))
+    log.trace("found zeromq transport addr:{0} using addr:{}".format(raw, addr))
```

Comment:
`raw` isn't defined, also you should be using `{0}` and `{1}`, not an empty bracket.

Reply:
I'm fixing this stuff. Coffee helps :)